.travis.yml

```
@@ -6,7 +6,7 @@ cache: bundler
 before_install:
   - gem update --system
   - gem --version
-  - gem install bundler --no-rdoc --no-ri
+  - gem install bundler
```

Oh, please change this to:

```suggestion
  - gem install bundler --no-document
```